tests/osqp_tester.h

```diff
@@ -1,5 +1,11 @@
 // Utilities for testing
 
+#ifndef OSQP_TESTER_H
+#define OSQP_TESTER_H
+
+#define mu_assert(msg, pred) do { INFO(msg); REQUIRE(pred); } while((void)0, 0)
```

Comment:
A couple of declarations previously in `minunit.h` have been moved here, and `minunit.h` is gone. Already approved in `master`.